src/main/scala/org/apache/spark/mllib/optimization/tfocs/LinearOperator.scala

```diff
@@ -120,3 +120,41 @@ class TransposedDenseVectorToDVectorLinOp(@transient private val matrix: DMatrix
 
   override def t: LinearOperator[DenseVector, DVector] = new DenseVectorToDVectorLinOp(matrix)
 }
+
+/**
+ * Compute the products A * x' and b * x', where 'A' is a DMatrix, 'x' is a Vector, and 'b' is a
+ * Vector. The two products are returned together as a tuple.
+ *
+ * NOTE In matlab tfocs this functionality is implemented in linop_stack.m.
+ * @see [[https://github.com/cvxr/TFOCS/blob/master/private/linop_stack.m]]
+ */
+class LinopMatrixVector(private val A: DMatrix, private val b: DenseVector)
```

Comment:
I haven't worried too much about the class names here because we will be reorganizing the naming conventions soon.